test/tests/samplers/distribute/tests

```diff
@@ -12,7 +12,7 @@
       valgrind = NONE
       detail = "that distributes samples in parallel and"
       python = 3
-      required_python_packages = 'pandas' # uncomment with upstream/python-3 is merged
+      required_python_packages = 'pandas matplotlib tornado'
```

Comment:
This can't be right, I have never heard of this package.  It is a "Python web framework and asynchronous networking library", I don't even know what those words mean.

Reply:
Well in the issue I posted, it was the cause of the test failure.

Reply:
It s an optional dependency of matplotlib and apparently some version of mpl needs it
https://matplotlib.org/stable/devel/dependencies.html

Reply:
ok looks like it's when people use Jupyter notebooks that they require it

Reply:
What do you think we should do? This isnt the way we want jupyter notebooks to fail. 
Should we add something to run_tests.py instead to check that mpl was installed with tornado on those?

Reply:
I don't understand how Juypter notebook can relate to this test failing. I don't think we should worry about supporting Jupyter notebooks.

Reply:
Ok I removed it. I m fine with waiting to see if someone else has this error pop up. It s not limiting anyone for sure anyway